core/scripts/vrfv2/testnet/main.go

```diff
@@ -822,7 +847,7 @@ func main() {
 		tx, err := linkToken.TransferAndCall(owner, coordinator.Address(), amount, b)
 		helpers.PanicErr(err)
 		fmt.Println("Funding sub", *subID, "TX", helpers.ExplorerLink(chainID, tx.Hash()))
```

Comment:
Could these log messages also be moved inside confirmTxMined to avoid duplicating them?

Reply:
I can do that, although not every TX log is generic. For example:

```
fmt.Println("Funding sub", *subID, "TX", helpers.ExplorerLink(chainID, tx.Hash()))
```

I would get rid of this and put a more generic one inside the helper method:

```
fmt.Println("Executing TX:", helpers.ExplorerLink(chainID, tx.Hash()))
```

Honestly just a matter of team preference.

Reply:
Just an idea, but maybe you can add a variadic string parameter that the confirmTxMined helper will log, prior to logging the `Executing TX` log at the end.

Reply:
fair enough

Reply:
```
Executing TX 0x7a977b9c715cf40ebb1dd2a49682b3f84d2ffa644a84c6551b9eb7e3424ab61a [Uncompressed public key: 12345678 Oracle address: 0x12345678]
```